packages/cashier/src/components/cashier-locked/cashier-locked.tsx

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import { observer } from 'mobx-react-lite';
 import EmptyState from 'Components/empty-state';
 import getMessage from './cashier-locked-provider';
+import { useDepositLocked } from '@deriv/hooks';
 import { useStore } from '@deriv/stores';
 
 const CashierLocked = () => {
+    const is_deposit_locked = useDepositLocked();
     const { client, modules } = useStore();
     const {
         account_status,
         accounts,
         current_currency_type,
-        is_deposit_lock: is_deposit_locked,
```

Comment:
@farzin-deriv I believe that it's ok if we will use `useDepositLocked` instead of `is_deposit_lock` from `ClientStore` because we show `CashierLocked` component first based on `is_cashier_locked` status and if we have `is_deposit_locked=true`, we show the user what was wrong with his account (in terms of `is_deposit_lock` status (without checking other extra logic)), and if he fixed his `is_cashier_locked` and `is_deposit_locked` to false and then again tries to reach cashier we again checking his deposit status, but now with extra logic as you mentioned and show him an appropriate message. Mmm maybe my explanation was not very clear, **but my conclusion is we can replace `is_deposit_lock` from `ClientStore` with `useDepositLocked` hook**

Reply:
@george-usynin-binary Thank you for your explanation buddy 🌹 🙇🏻